src/Components/HeaderPanel/HeaderPanel.tsx

```diff
@@ -1,22 +1,22 @@
-import React, { ReactNode } from "react";
+import React from "react";
 import Nav from "components/Nav/Nav";
 
 import bemCssModule from "bem-css-modules";
 import { default as HeaderPanelStyles } from "./HeaderPanel.module.scss";
 import Header from "components/Header/Header";
 
 interface Props {
-  actionButton: ReactNode;
+  titleButton: string;
 }
 
 const style = bemCssModule(HeaderPanelStyles);
 
-const HeaderPanel: React.FC<Props> = ({ actionButton }) => {
+const HeaderPanel: React.FC<Props> = ({ titleButton }) => {
   return (
     <div className={style()}>
       <Header />
       <Nav />
-      {actionButton}
+      <button className={style("button")}>{titleButton}</button>
```

Comment:
It is ok, but in the future you may need additional props to button like icon, filled/outlined style, etc. and then it will be much harder to implement than pass single button component from the parent. So it's ok, but this approach is very limited.